ts_src/payments/p2wsh.ts

```diff
@@ -18,6 +19,15 @@ function stacksEqual(a: Buffer[], b: Buffer[]): boolean {
   });
 }
 
+function chunkHasUncompressedPubkey(chunk: StackElement): boolean {
+  if (Buffer.isBuffer(chunk) && chunk.length === 65) {
+    if (ecc.isPoint(chunk)) return true;
+    else return false;
+  } else {
+    return false;
+  }
```

Comment:
I think it could be even simpler.
```suggestion
  if (Buffer.isBuffer(chunk) && chunk.length === 65 && ecc.isPoint(chunk)) {
    return true;
  }
  return false;
```